R/as_html.R

```diff
@@ -61,7 +62,8 @@ as_html <- function(x,
                     class_tr = NULL,
```

Comment:
I am a bit new to this function, sorry. What are these class inputs?

Reply:
These arguments allow users to specify an HTML class with HTML specs for `table`/`tr` (table row)/`th` (table header) tags if they want (some already exist, but they can also be created).